assets/css/disruption_calendar.css

```
@@ -1,23 +1,47 @@
-.fc-event.route-Red {
+.fc-event.approved-route-Red {
   background-color: #da291c;
 }
 
-.fc-event.route-Blue {
+.fc-event.pending-route-Red {
+  background-color: #f4bfbb;
+}
+
+.fc-event.approved-route-Blue {
   background-color: #003da5;
 }
 
-.fc-event.route-Orange {
+.fc-event.pending-route-Blue {
+  background-color: #b3c5e4;
+}
+
+.fc-event.approved-route-Orange {
   background-color: #ed8b00;
 }
 
-.fc-event.route-Mattapan {
+.fc-event.pending-route-Orange {
+  background-color: #f9d6a6;
+}
+
+.fc-event.approved-route-Mattapan {
   background-color: #da291c;
 }
 
-.fc-event[class*="route-Green"] {
+.fc-event.pending-route-Mattapan {
+  background-color: #f4bfbb;
+}
+
+.fc-event[class*="approved-route-Green"] {
   background-color: #00843d;
 }
 
-.fc-event[class*="route-CR"] {
+.fc-event[class*="pending-route-Green"] {
+  background-color: #cce6d8;
+}
+
+.fc-event[class*="approved-route-CR"] {
   background-color: #80276c;
 }
+
+.fc-event[class*="pending-route-CR"] {
+  background-color: #d98fd3;
+}
```

suggestion: needs a trailing newline
```suggestion
}

```